src/frame/sdk/tests/DocumentApi.test.ts

```diff
@@ -435,6 +435,28 @@ describe("DocumentApi", () => {
                 () => fail("Should not call create when any user or group could not be found")
             );
         });
+
+        it("fails if there is no one to encrypt to.", (done) => {
+            spyOn(UserApiEndpoints, "callUserKeyListApi").and.returnValue(
+                Future.of({
+                    result: [],
+                })
+            );
+            spyOn(GroupApiEndpoints, "callGroupKeyListApi").and.returnValue(
+                Future.of({
+                    result: [],
+                })
+            );
+
+            DocumentApi.encryptLocalDocument("doc key", new Uint8Array([88, 73, 92]), "", [], [], false).engage(
+                (e) => {
+                    expect(e.message).toBeString();
+                    expect(e.message).toBe("Failed to create document due to no users or groups to share with.");
+                    done();
+                },
+                () => fail("Should not call create when no users or groups could be found")
+            );
+        });
```

Comment:
Would a test with grants and `false` on `grantToAuthor` make sense?

Reply:
added.